Code/Framework/AzToolsFramework/Tests/PropertyIntSpinCtrlTests.cpp

```diff
@@ -47,4 +47,71 @@ namespace UnitTest
     {
         this->HandlerMinMaxLessLimit_ModifyHandler_ExpectSuccessAndValidLessLimitToolTipString();
     }
+
+    struct PropertyEditorHandler
+        : public AzToolsFramework::PropertyEditorGUIMessages::Bus::Handler
+    {
+        PropertyEditorHandler()
+        {
+            AzToolsFramework::PropertyEditorGUIMessages::Bus::Handler::BusConnect();
+        }
+
+        ~PropertyEditorHandler()
+        {
+            AzToolsFramework::PropertyEditorGUIMessages::Bus::Handler::BusDisconnect();
+        }
+
+        // AzToolsFramework::PropertyEditorGUIMessages::Bus overrides ...
+        void RequestWrite([[maybe_unused]] QWidget* editorGUI) override
+        {
+            m_requestWriteCallCount++;
+        }
+
+        void RequestRefresh([[maybe_unused]] PropertyModificationRefreshLevel level) override
+        {
+        }
+
+        void AddElementsToParentContainer(
+            [[maybe_unused]] QWidget* editorGUI,
+            [[maybe_unused]] size_t numElements,
+            [[maybe_unused]] const InstanceDataNode::FillDataClassCallback& fillDataCallback) override
+        {
+        }
+
+        void RequestPropertyNotify([[maybe_unused]] QWidget* editorGUI) override
+        {
+        }
+
+        void OnEditingFinished([[maybe_unused]]QWidget* editorGUI) override
+        {
+            m_onEditingFinishedCallCount++;
+        }
+
+        int m_requestWriteCallCount = 0;
+        int m_onEditingFinishedCallCount = 0;
+    };
+
+    TYPED_TEST(PropertySpinCtrlFixture, SpinBoxWidget_valueChanged_InvokesPropertyEditorGUIMessages)
```

Comment:
The new convention is to not use "_" in the test name because it's a reserved character of gtest.

SpinBoxWidget_valueChanged_InvokesPropertyEditorGUIMessages -> SpinBoxWidgetValueChangedInvokesPropertyEditorGUIMessages